homeassistant/components/islamic_prayer_times/__init__.py

```diff
@@ -67,7 +72,7 @@ def get_new_prayer_times(self):
         )
         return calc.fetch_prayer_times()
 
-    async def async_schedule_future_update(self):
+    async def async_schedule_future_update(self, midnight_dt) -> None:
```

Comment:
And add a type hint here also
```suggestion
    async def async_schedule_future_update(self, midnight_dt: datetime) -> None:
```

Reply:
Done